lib/provider/ui/index.tsx

```diff
@@ -10,6 +10,7 @@ import { themeAtom } from "~/lib/theme-atom";
 
 import { Header } from "./header";
 import { Windows } from "./windows";
+import { store } from "./windows/solitaire/jotai";
```

Comment:
What do we need store for? We shouldn't use global store like this because it would be shared between server-rendered routes and potentially cause conflicts or security issues

Reply:
Instead please use pure atoms or useStore hook if store is unavoidable